### PR TITLE
docs: remove unnecessary incorrect front matter labels

### PR DESCRIPTION
**Pull request by imatwawana**
Remove unnecessary front matter labels from shared content files.
Also fixing preview note for email sharing.

### CODE REVIEW

File: docs/sources/shared/visualizations/change-axis-link.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
 ---
 labels:
   products:
+    - cloud
     - enterprise
     - oss
```
Comment (imatwawana) [inline docs/sources/shared/visualizations/change-axis-link.md:6]:
```suggestion
```

File: docs/sources/shared/visualizations/connect-null-values.md [hunk 2]
```diff
@@ -1,4 +1,9 @@
 ---
+labels:
+  products:
+    - cloud
+    - enterprise
+    - oss
```
Comment (imatwawana) [inline docs/sources/shared/visualizations/connect-null-values.md:6]:
```suggestion
```

File: docs/sources/shared/visualizations/legend-mode.md [hunk 3]
```diff
@@ -1,6 +1,7 @@
 ---
 labels:
   products:
+    - cloud
     - enterprise
     - oss
```
Comment (imatwawana) [inline docs/sources/shared/visualizations/legend-mode.md:6]:
```suggestion
```

File: docs/sources/shared/visualizations/stack-series-link.md [hunk 4]
```diff
@@ -1,6 +1,7 @@
 ---
 labels:
   products:
+    - cloud
     - enterprise
     - oss
```
Comment (imatwawana) [inline docs/sources/shared/visualizations/stack-series-link.md:6]:
```suggestion
```

File: docs/sources/shared/visualizations/tooltip-mode.md [hunk 5]
```diff
@@ -1,6 +1,7 @@
 ---
 labels:
   products:
+    - cloud
     - enterprise
     - oss
```
Comment (imatwawana) [inline docs/sources/shared/visualizations/tooltip-mode.md:6]:
```suggestion
```

**Review by nmarrs — APPROVED** [review]

File: docs/sources/dashboards/dashboard-public/index.md [hunk 6]
```diff
@@ -6,7 +6,6 @@ labels:
   products:
     - cloud
     - enterprise
-    - oss
```
Comment (juanicabanas) [inline docs/sources/dashboards/dashboard-public/index.md:9]: we don't mention oss anymore?

**Review by juanicabanas — COMMENTED** [review]
left a little comment

File: docs/sources/dashboards/dashboard-public/index.md (hunk 6 above)
Comment (evictorero) [reply docs/sources/dashboards/dashboard-public/index.md:9]: Public dashboards are in OSS and they are in public preview.
Email sharing is for Cloud and Enterprise; this particular feature is in private preview (not public preview) like we have below. Can we change that too?

File: docs/sources/dashboards/dashboard-public/index.md [hunk 7]
```diff
@@ -6,7 +6,6 @@ labels:
   products:
     - cloud
     - enterprise
```
Comment (imatwawana) [inline docs/sources/dashboards/dashboard-public/index.md:8]:
```suggestion
    - enterprise
    - oss
```

**Comment by imatwawana** [comment]
@juanicabanas & @evictorero - Thanks for the review! I reverted the change to take oss out and updated the preview note for email sharing.

**Pull request merged**